datasets/submission/submission-guide.qmd

```diff
@@ -5,13 +5,22 @@ title: Submission guide
 This guide contains instructions on how to upload data to BigPicture.
 We will take you through the process step by step. 
 
+0.  [Test submission](#test-submission)
 1.  [Sign and send Terms of Use form](#send-terms-of-use-form)
 2.  [Install the sda-cli tool](#install-the-sda-cli-tool)
 3.  [Download the encryption key](#download-the-encryption-key)
 4.  [Encrypt the files](#encrypt-the-files)
 5.  [Submit your files](#submit-your-files)
 6.  [Notify us](#notify-us)
 
+## Test submission {#test-submission}
+
+We highly recommend that you first do a small submission to our [staging](https://staging-login.bp.nbis.se/) environment, this way we can avoid requiring you to re-upload terabytes of data.
+
+We recommend that you upload two images and generate metadata for them according to the preparation guide, please include all XML files don't forget the `PRIAVET` folder or the `LANDING_PAGE` (in case you want one of those).
```

Comment:
```suggestion
We recommend that you upload two images and generate metadata for them according to the preparation guide, please include all XML files don't forget the `PRIVATE` folder or the `LANDING_PAGE` (in case you want one of those).
```